applications/openrollup-mvp-phase-1.md

```diff
@@ -0,0 +1,179 @@
+- **Website:** https://openrollup.org/ (not yet established)
+
+### Legal Structure
+
+- **Registered Address:**  not yet available
+- **Registered Legal Entity:** not yet available
+
+### Team's experience
+
+Our team  have much experience in applied cryptography, blockchain research, and software development. 
+Chris has been a software engineer for over many years in the gaming industry and has led the development of various multiplayer online games. Recently he develop a hybrid consensus of Progpow and AuthorityRound [https://github.com/chrisicen/openethereum-unity](https://github.com/chrisicen/openethereum-unity). He is currently devoted to the development of off-chain ZK scalability solution for blockchain.
+MJewzk is a blockchain development engineer and has been involved in the development of some blockchain and web3 projects.
+
+### Team Code Repos
+
+- https://github.com/openrollup-zk
+- https://github.com/chrisicen
+- https://github.com/MJewzk
+
+### Team LinkedIn Profiles (if available)
+
+not yet available
+
+## Development Status :open_book:
+
+We've been working on techniques for zk-rollup, and verifiable computation for some time, including projects like zksync, halo2, plonky2, winterfell, hermez, and related papers and articles. We are starting to implement the rollup part.
+
+## Development Roadmap :nut_and_bolt:
+
+### Overview
+
+- **Total Estimated Duration:** 2.7 months
+- **Full-Time Equivalent (FTE):** 1.5 FTE
+- **Total Costs:** 48,600 USD).
+
+### Milestone 1 — Implement Substrate Modules
+
+- **Estimated duration:** 2.7 month
+- **FTE:**  1.5
+- **Costs:** 48,600 USD
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| 0a. | License | Apache 2.0 |
+| 0b. | Documentation | We will provide both **inline documentation** of the code and a basic **tutorial** that explains how a user can spin up one of our Substrate nodes and send test transactions, which will show how the new functionality works. |
+| 0c. | Testing Guide | Core functions will be fully covered by unit tests to ensure functionality and robustness. In the guide, we will describe how to run these tests. |
+| 0d. | Docker | We will provide a Dockerfile(s) that can be used to test all the functionality delivered with this milestone. |
+| 0e. | Article | We will publish an **article**/workshop that explains what was done/achieved as part of the grant. |
+| 1.  | Setup | We provide a repository that forms the basis of our deliverables.|
+| 2.  | Core Types | We provide the core data types, as specified in Project Details. |
+| 3.  | Verifier trait and mock verifier | We provide the verifier trait suitable for general apps and a mock verifier that always pass the zk-verification. |
+| 4.  | Pallet functions | We provide the functionalities(zkapp registration/user deposit/user exit/user full exit/zkapp batch submit/zkapp management/zkapp info api) which support currencies/tokens/NFTs as specified in Project Details. |
```

Comment:
This both sounds to me like you're describing the pallet itself. If that's the case, feel free to merge the two deliveries into 1.

Reply:
hi, @takahser , I would like our pallet to support various zkvms, and the recent revision added miden verifier support in this delivery,  so it's a little clearer to keep a separate delivery.